events/helper/outbound_connection.go

```diff
@@ -0,0 +1,20 @@
+package helper
+
+import (
+	"net"
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(OutboundConnection)
+
+func OutboundConnection(h events.Helper) error {
+	conn, err := net.Dial("udp", "8.8.8.8:53")
```

Comment:
It would be nice to find a neutral domain/ip (eg. `example.net`) to avoid disturbing a real service like `8.8.8.8`.